qemu/tests/win_virtio_single_driver.py

```diff
@@ -0,0 +1,166 @@
+import re
+import logging
+from autotest.client.shared import error
+from virttest import utils_misc
+
+
+@error.context_aware
+def run(test, params, env):
+    """
```

Comment:
As we discuss, pls update the description to let user know what's the diff with driver_update